app/src/main/java/by/itechart/android/ui/screen/MainActivity.kt

```diff
@@ -1,6 +1,42 @@
 package by.itechart.android.ui.screen
 
+import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
+import androidx.navigation.findNavController
 import by.itechart.android.R
+import by.itechart.android.ext.hide
+import by.itechart.android.ext.show
+import kotlinx.android.synthetic.main.activity_main.*
+import kotlinx.android.synthetic.main.activity_main.menuOpenImageView
+import kotlinx.android.synthetic.main.view_navigation_drawer.*
 
-class MainActivity : AppCompatActivity(R.layout.activity_main)
+class MainActivity : AppCompatActivity(R.layout.activity_main) {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        menuOpenImageView.setOnClickListener { mainDrawerLayout.openDrawer(sideMenuNavigation) }
+        menuCloseImageView.setOnClickListener { mainDrawerLayout.closeDrawer(sideMenuNavigation) }
+
+        val navController = findNavController(R.id.primaryNavHostFragment)
+        navController.addOnDestinationChangedListener { _, destination, _ ->
+            if (bottomNavView.menuIds.contains(destination.id)) {
+                bottomNavView.show()
```

Comment:
need to disable navigation drawer (it is possible to open nav drawer via swipe to right)